tests/test_data_store_merge_platforms.py

```diff
@@ -0,0 +1,680 @@
+            )
+            contacts_after_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor_2.sensor_id).all()
+            )
+            comments_after_merge = (
+                self.store.session.query(Comment)
+                .filter(Comment.platform_id == platform.platform_id)
+                .all()
+            )
+            assert len(sensors_after_merge) == 2
+            assert len(states_after_merge) == 2
+            assert len(contacts_after_merge) == 2
+            assert len(comments_after_merge) == 1
+
+            # Assert that sensor_2 moved to Platform
+            assert sensors_after_merge[1] == sensor_2
+
+
+class UpdatePlatformIDsTestCase(TestCase):
+    def setUp(self):
+        self.store = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+        self.store.initialise()
+        self.parser_name = "Test Importer"
+        self.current_time = datetime.utcnow()
+
+        with self.store.session_scope():
+            self.change_id = self.store.add_to_changes("TEST", datetime.utcnow(), "TEST").change_id
+            self.nationality = self.store.add_to_nationalities(
+                "test_nationality", self.change_id
+            ).name
+            self.platform_type = self.store.add_to_platform_types(
+                "test_platform_type", self.change_id
+            ).name
+            self.sensor_type = self.store.add_to_sensor_types(
+                "test_sensor_type", self.change_id
+            ).name
+            self.comment_type = self.store.add_to_comment_types("test_type", self.change_id)
+            self.privacy = self.store.add_to_privacies("test_privacy", 0, self.change_id)
+            self.privacy_id = self.privacy.privacy_id
+            self.file = self.store.get_datafile("test_file", "csv", 0, "HASHED", self.change_id)
+            self.file.measurements[self.parser_name] = dict()
+
+            self.platform = self.store.get_platform(
+                platform_name="Test Platform",
+                nationality=self.nationality,
+                platform_type=self.platform_type,
+                privacy=self.privacy.name,
+                change_id=self.change_id,
+            )
+            self.sensor = self.platform.get_sensor(
+                self.store, "gps", self.sensor_type, change_id=self.change_id
+            ).sensor_id
+            self.platform_2 = self.store.get_platform(
+                platform_name="Test Platform 2",
+                nationality=self.nationality,
+                platform_type=self.platform_type,
+                privacy=self.privacy.name,
+                change_id=self.change_id,
+            )
+
+            self.store.session.expunge(self.file)
+            self.store.session.expunge(self.comment_type)
+            self.store.session.expunge(self.platform)
+            self.store.session.expunge(self.platform_2)
+
+    def test_update_platform_ids_except_media(self):
```

Comment:
I think this needs renaming now that it includes media

Reply:
Thanks for pointing out. I updated out of date docstrings and comments: https://github.com/debrief/pepys-import/pull/740/commits/94ce9d8240683472553fd09c225987e45e4f70b3